examples/sph-mpi/sph-main-mpi.cpp

```diff
@@ -409,10 +415,12 @@ void densityPressureHydroForce(AutoPasContainer &sphSystem, MPI_Comm &comm, cons
 }
 
 void printConservativeVariables(AutoPasContainer &sphSystem, MPI_Comm &comm) {
+  using namespace autopas::utils::ArrayMath::literals;
+
   std::array<double, 3> momSum = {0., 0., 0.};  // total momentum
   double energySum = 0.;                        // total energy
   for (auto it = sphSystem.begin(autopas::IteratorBehavior::owned); it.isValid(); ++it) {
-    momSum = autopas::utils::ArrayMath::add(momSum, autopas::utils::ArrayMath::mulScalar(it->getV(), it->getMass()));
+    momSum = momSum + (it->getV() * it->getMass());
```

Comment:
you could also implement [assignment operators](https://en.cppreference.com/w/cpp/language/operator_assignment)